design-toolkit/components/package.json

```diff
@@ -0,0 +1,153 @@
+{
+  "name": "@accelint/design-toolkit",
+  "version": "0.0.0",
+  "private": false,
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gohypergiant/standard-toolkit/design-toolkit"
+  },
+  "type": "module",
+  "files": ["dist/**"],
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      },
+      "default": "./dist/index.js"
+    },
+    "./avatar": {
+      "import": {
+        "types": "./dist/components/avatar/index.d.ts",
+        "default": "./dist/components/avatar/index.js"
+      },
+      "default": "./dist/components/avatar/index.js"
+    },
+    "./badge": {
+      "import": {
+        "types": "./dist/components/badge/index.d.ts",
+        "default": "./dist/components/badge/index.js"
+      },
+      "default": "./dist/components/badge/index.js"
+    },
+    "./button": {
+      "import": {
+        "types": "./dist/components/button/index.d.ts",
+        "default": "./dist/components/button/index.js"
+      },
+      "default": "./dist/components/button/index.js"
+    },
+    "./classification-banner": {
+      "import": {
+        "types": "./dist/components/classification-banner/index.d.ts",
+        "default": "./dist/components/classification-banner/index.js"
+      },
+      "default": "./dist/components/classification-banner/index.js"
+    },
+    "./floating-button": {
+      "import": {
+        "types": "./dist/components/floating-button/index.d.ts",
+        "default": "./dist/components/floating-button/index.js"
+      },
+      "default": "./dist/components/floating-button/index.js"
+    },
+    "./icon-button": {
+      "import": {
+        "types": "./dist/components/icon-button/index.d.ts",
+        "default": "./dist/components/icon-button/index.js"
+      },
+      "default": "./dist/components/icon-button/index.js"
+    },
+    "./toggle-icon-button": {
+      "import": {
+        "types": "./dist/components/toggle-icon-button/index.d.ts",
+        "default": "./dist/components/toggle-icon-button/index.js"
+      },
+      "default": "./dist/components/toggle-icon-button/index.js"
+    },
+    "./tooltip": {
+      "import": {
+        "types": "./dist/components/tooltip/index.d.ts",
+        "default": "./dist/components/tooltip/index.js"
+      },
+      "default": "./dist/components/tooltip/index.js"
+    },
+    "./icons": {
+      "import": {
+        "types": "./dist/icons/index.d.ts",
+        "default": "./dist/icons/index.js"
+      },
+      "default": "./dist/icons/index.js"
+    },
+    "./styles": {
+      "import": {
+        "default": "./dist/index.css"
+      },
+      "default": "./dist/index.css"
+    }
+  },
+  "scripts": {
+    "bench": "pnpm vitest bench --run --dir src",
+    "build": "pnpm tsup && pnpm build:css",
+    "build:css": "npx @tailwindcss/cli -i ./src/index.css -o ./dist/styles.css",
```

Comment:
Small nit: `pnpm dlx` is the equivalent command for `npx`. Alternatively we should maybe just define `@tailwindcss/cli` as a devDependency and call it like so `pnpm @tailwind/cli ...` so that we are not downloading anything during pipeline runs